frame/contracts/src/exec.rs

```diff
@@ -104,6 +104,19 @@ pub trait Ext: sealing::Sealed {
 		allows_reentry: bool,
 	) -> Result<ExecReturnValue, ExecError>;
 
+	/// Execute code in the current frame.
+	///
+	/// Returns the original code size of the called contract.
+	///
+	/// # Return Value
+	///
+	/// Result<(ExecReturnValue, CodeSize), (ExecError, CodeSize)>
```

Comment:
Please remove this stale comment.

Reply:
why is it stale?

Reply:
Because it is wrong. The function doesn't return the size. The function you copied the docs from used to do that. Their comment is also stale.

Reply:
oh yeah, that is true